### DISCUSSION TITLE
EAF browser isn't working when running emacs in daemon mode.

### DISCUSSION THREAD

**Discussion by jabuxas** [emacs-eaf/emacs-application-framework#1149] [Q&A]
I have two problems at the moment with EAF,

If I run the gui version, the browser works, but keybindings don't. I can't even press `M-x` to run any commands.
If I run the daemon version, the commands work, but, the buffer stays black like so: 
![image](https://github.com/emacs-eaf/emacs-application-framework/assets/94939040/6fee0ecd-c75c-4a4f-93e7-37408c7759c7)

It might be related to #1140,

I just installed it, looked through FAQ and Wiki and found nothing that could help me.

I'm running Hyprland with Gentoo.

Here is my config: https://bpa.st/2DFA

Any ideas? All help is appreciated

**Comment by manateelazycat** [comment]
Please use Gnome or KDE to test.

**Comment by jabuxas** [comment]
Why would that be a problem? It works when I open emacs as not a daemon
